Core/Infrastructure/Http/SignUp/SignUpRepository.cs

```diff
@@ -16,19 +16,11 @@ public SignUpRepository(IAPIManager apiManager)
         {
         }
 
-        public Task RegisterDispatcher(DispatcherCompanyDM dispatcherCompany, RequestPriority priority)
+        public Task Register(CompanyApplicationDM companyApplication, RequestPriority priority)
         {
-            var company = Mapper.Map<DispatcherCompanyEM>(dispatcherCompany);
+            var companyApplicationEM = Mapper.Map<CompanyApplicationEM>(companyApplication);
```

Comment:
Название переменной companyApplicationEM не корректно. Если уже есть companyApplication, то можно переименовать ее на domainCompanyApplication

Reply:
Мне кажется, что domain больше относится к контексту серверной части. Может, лучше назвать переменную companyApplicationDTO? DTO - Data transfer object.

Reply:
Одно из правил именования переменных "using names based on a parameter’s meaning rather than the parameter’s type" это значит, что некорректно использовать "EM", "DTO" и т.п. обозначение и имени переменной.

Reply:
Да, согласен.